File: Start-Drive.API/Start-Drive.API/Models/CourseRoundsModel.cs
namespace Start_Drive.API.Models
{
    public class CourseRoundsModel
    {
        public int Id { get; set; }
        public RegisterSchool DrivingSchool { get; set; }
        public int DrivingSchoolId { get; set; }

        public string Name { get; set; }
        public DateTime From { get; set; }
        public DateTime To { get; set; }
        public List<CourseRoundsStudentsId> StudentsThisRoundCourse { get; set; }
    }
}

File: Start-Drive.UI/src/app/models/course-rounds.model.ts
import { CourseRoundsStudentsId } from "./course-rounds-students-id.model";
import { RegisterSchool } from "./register-school.model";

export interface CourseRoundsModel{
  id?: number,
  drivingSchool?: RegisterSchool,
  drivingSchoolId?: number,

  name: string,
  from: Date,
  to: Date,
  studentsThisRoundCourse: CourseRoundsStudentsId[]
}
